lib/util/decimalize-member.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Applies a decimal transform on an object's member
+ *
+ * @param {String} prop Property on {this} to transform
+ * @this {Object} on which to apply decimal transformation
+ */
+
+export default function decimalizeMember (prop) {
+  this[prop] = (Math.round(Math.max(this[prop], 0) * 100) / 100).toFixed(2);
```

Comment:
does this need to account for zero decimal currencies?

Reply:
We might consider that for another change... it would add '.00' to a ZDC, which isn't proper for display, but the underlying values are sound.